dangerfile2.js

```diff
@@ -0,0 +1,160 @@
+// inspire by reacts dangerfile
```

Comment:
@mbrookes has this file been added by accident?

Reply:
It appears so